board/download_dependencies.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+#
+# This script download OpenBLAS dependecy for Armadillo and cross-compile them against
+# required architecture.
+#
+board_name=$1
+
+echo "Download and build boost as a static lbrary"
+wget --no-check-certificate \
+    "https://dl.bintray.com/boostorg/release/1.75.0/source/boost_1_75_0.tar.gz" && \
+    tar xvzf boost_1_75_0.tar.gz && \
+    rm -f boost_1.75.0.tar.gz && \
```

Comment:
No need for a line wrap here?  Also, I think the `echo` is wrong---we don't need to build Boost, since we're only using headers from it.